pkg/datastore/datastore_with_txcontext_test.go

```diff
@@ -83,7 +85,7 @@ func testTxCrud(t *testing.T, ds datastore.DataStore, ctx context.Context, user1
 		queryResult := &AppUser{Id: record.Id}
 		err = ds.Find(txCtx, queryResult)
 		assert.NoError(err)
-		assert.Equal(record, queryResult)
+		cmp.Equal(record, &queryResult, cmpopts.IgnoreFields(AppUser{}, "CreatedAt", "UpdatedAt", "DeletedAt"))
```

Comment:
This doesn't seem to assert only return bool

Reply:
Is it easy to figure out the diff on failures or do you need debugger with assert.True?